x-pack/plugins/console_extensions/spec/overrides/xpack.ml.get_records.json

```diff
@@ -0,0 +1,16 @@
+{
+  "xpack.ml.get_records": {
+    "data_autocomplete_rules": {
+      "desc": "__flag__",
```

Comment:
In the [xpack.info endpoint](https://github.com/elastic/kibana/pull/19928/files#diff-6fa7381b3516d4319b4cbdf792d7cf00R9), `__flag__` is used within a URL parameter, and autocompleter will suggest `true` or `false` as values for the param. In this case, the autocompleter doesn't suggest these values and instead autofills the value as the string `"__flag__"`.

Reply:
Fixed this here and elsewhere.